Ds3/Helpers/Ds3ClientHelpers.cs

```diff
@@ -85,24 +68,40 @@ public class Ds3ClientHelpers : IDs3ClientHelpers
                     }
 
                     retriesLeft--;
-                    Thread.Sleep(this._jobWaitTime*1000*60);
+                    Thread.Sleep(_jobWaitTime * 1000 * 60);
                 }
             } while (jobResponse == null);
 
             if (helperStrategy == null)
             {
-                helperStrategy = new WriteRandomAccessHelperStrategy(this._retryAfter, withAggregation);
+                helperStrategy = new WriteRandomAccessHelperStrategy(_retryAfter, ds3WriteJobOptions != null && (ds3WriteJobOptions.Aggregating ?? false));
```

Comment:
If this uses the change from above, you won't have to check for null or assign a default value here.